sources/packages/backend/libs/test-utils/src/factories/application.ts

```diff
@@ -101,6 +101,7 @@ export async function saveFakeApplicationDisbursements(
     applicationStatus?: ApplicationStatus;
     offeringIntensity?: OfferingIntensity;
     createSecondDisbursement?: boolean;
+    currentAssessmentInitialValues?: Partial<StudentAssessment>;
```

Comment:
how about adding this new option in the comment too?

Reply:
Done!